notifiers/providers/victorops.py

```diff
@@ -77,7 +79,15 @@ class VictorOps(Provider):
     def _prepare_data(self, data: dict) -> dict:
         annotations = data.pop("annotations", {})
         for annotation, value in annotations.items():
-            data[annotation] = value
+            if re.match(r"vo_annotate.[usi].", annotation):
```

Comment:
there's no need for this, this is the advantage of using schema. If you set `additionalProperties` as False under `annotations`, this will be enforced at the schema load

Reply:
that's what i wrote in your previous comment, it doesn’t  enforce it 

> I'm updating the "title" (I hope this is what you meant).
> I also checked the json-schema documentation and looks like this schema won’t prevent someone from adding an annotation field that is not supported by victorops (this feature will be new I guess and is not yet documented _"Unevaluated Properties"_).
> 
> I will add some logic in _prepare_data to fail if it doesn't match the support strings.
> Tell me if this sounds good to you.

Reply:
Got it :) :+1: